src/gov/nist/javax/sip/parser/SIPMessageListener.java

```diff
@@ -45,7 +45,7 @@
      */
     public void processMessage(SIPMessage msg) throws Exception;
 
-	public void sendSingleCLRF() throws Exception;
+	public void sendSingleCRLF() throws Exception;
```

Comment:
even though this seems like a clear typo, i'm concerned this would break backwords compatibility... 🤔 

Maybe a default implementation here for the new one invoking the old one would help us get the best of both worlds... 

@jaimecasero thoughts?

Reply:
this interface is not part of JAIN-SIP spec, and is not a boundary with the App. Is an internal API to allow IO layer to contact Parsing layer. If we change this interface, we will need to cover all the neccesary changes tin IO and parsing layer (XPipelineParsers,ConnOrientedMsgChannel,TCPMsgChannel,TLSMsgChannel...).

not sure is worth the change, but we could do it if the PR is comprehensive and tests results are ok

Reply:
I would suggests to handle in different PRs, as the original concern about double CRLF on TCP closing seems valid. We could potentially merge this asap, and then think about the typo...